content/docs/en/elements/components/tab-view.md

```diff
@@ -1,20 +1,20 @@
 ---
 title: TabView
 apiRef: https://docs.nativescript.org/api-reference/classes/_ui_tab_view_.tabview
-contributors: [MisterBrownRSA, rigor789, eddyverbruggen]
+contributors: [MisterBrownRSA, rigor789, eddyverbruggen, ikoevska]
 ---
 
-The TabView component allows you to display content grouped into tabs.
+`<TabView>` is a navigation component that shows content grouped into tabs and lets users switch between tabs.
```

Comment:
Most of the other pages have a very similar structure to the first sentence "The XY component is/allows etc.". I like the addition of the code formatting on `<TabView>` but I think something like 
> The `<TabView>` component is a navigation component that shows...
would work better! What do you think? I'm open for discussion, just want to keep the docs consistent across all pages

Reply:
I see what you mean and  completely agree. However, I'd like us for all to move towards "X is a blah-blah component that does blah blah." We can either start pushing this change gradually now, or we can do it in bulk at some point. Let me know what you think.

Reply:
Either works, and that's alright, as long as we keep it consistent for the other pages too! Gradually pushing the change is fine.